assets/RNAseq_report.Rmd

```diff
@@ -750,7 +750,40 @@ columns <- c("Sample",
             "Cutadapt_mqc.generalstats.cutadapt.percent_trimmed",
             "STAR_mqc.generalstats.star.uniquely_mapped_percent",
             "featureCounts_mqc.generalstats.featurecounts.percent_assigned")
-df_DE_selected = df_DE[,columns]
+df_DE_selected <- df_DE[,columns]
+
+# The following gets rid of multiQC entries that are spread across two rows, one ending with -fw (see https://github.com/qbic-pipelines/rnadeseq/issues/114)
+for (r in 2:nrow(df_DE_selected)) {
```

Comment:
It's not computationally efficient to iterate with for loops over a dataframe. Is there any other way you can envision to remove those samples by applying filtering to the dataframe e.g. with dplyr and grouping by sample id?

Reply:
Done!